generated-platform-project/quarkus-universe/bom/pom.xml

```diff
@@ -25712,22 +25496,22 @@
       <dependency>
         <groupId>org.springframework</groupId>
         <artifactId>spring-aop</artifactId>
-        <version>5.3.18</version>
+        <version>5.3.19</version>
```

Comment:
The same about some of these srg.springframework ones. It's not a blocker but to clarify whether we can remove them in the future.